modules/installation-overview.adoc

```diff
@@ -26,18 +26,40 @@ These two basic types of {product-title} clusters are frequently called
 installer-provisioned infrastructure clusters and user-provisioned
 infrastructure clusters.
 
+Both types of clusters have the following characteristics:
+
+* Highly available infrastructure with no single points of failure
```

Comment:
Well, you could ask for a single control-plane node, or some other non-HA thing, with either approach.  Maybe append "possible" or some such?

Reply:
The documented minimum number of control plane nodes is 3. Please file a bug and get PM to agree if you want me to admit otherwise. :smile_cat:

Reply:
Uh, maybe you decide to run three control-plane VMs on a single physical host?  Stretching, but I'd rather emphasize HA-by-default, and avoid claiming unqualified HA regardless of anything admins do to shoot themselves in the foot ;)

Reply:
Lol, I'll add a note to the UPI topic to say that you should use different physical hosts for the machines, too.

Reply:
> Lol, I'll add a note to the UPI topic to say that you should use different physical hosts for the machines, too.

I'm less concerned about specific holes than I am about the fact that, no matter how hard we try, there will probably still be holes ;).  So I'd rather have some wiggle in blanket HA claims.

Reply:
Is this enough wiggle room?
> Highly available infrastructure with no single points of failure is available
by default

Reply:
yup, works for me :)